src/components/common/YourCompany/ResolutionDate.vue

```diff
@@ -0,0 +1,229 @@
+              and 90 days in the future. Make certain that this is the correct date as it cannot be easily
+              corrected afterwards.
+            </p>
+            <DatePickerShared
+              title="Resolution Date"
+              nudge-right="80"
+              nudge-top="15"
+              :minDate="startDateMinStr"
+              :maxDate="startDateMaxStr"
+              @emitDate="onOkClicked($event)"
+            />
+          </div>
+          <div class="float-right mb-2">
+            <v-btn large color="primary" id="start-done-btn" class="mr-2" @click="onDoneClicked()">
+              <span>Done</span>
+            </v-btn>
+            <v-btn large outlined color="primary" id="start-cancel-btn" @click="onCancelClicked()">
+              <span>Cancel</span>
+            </v-btn>
+          </div>
+        </template>
+
+        <span class="mt-n2 mr-n3 float-right">
+          <!-- Correct changes button -->
+          <v-btn v-if="isCorrectionFiling && !onEditMode && !isCorrected" text color="primary"
+            id="start-changes-btn" @click="onChangeClicked()">
+            <v-icon small color="primary">mdi-pencil</v-icon>
+            <span>{{ editLabel }}</span>
+          </v-btn>
+
+          <!-- Undo changes buttons -->
+          <v-btn v-if="isCorrectionFiling && !onEditMode && isCorrected"
+          text color="primary" id="start-undo-btn" @click="onUndoClicked()">
+            <v-icon small>mdi-undo</v-icon>
+            <span>Undo</span>
+          </v-btn>
+          <v-menu v-if="isCorrectionFiling && !onEditMode && isCorrected"
+            offset-y left nudge-bottom="4" v-model="dropdown">
+            <template v-slot:activator="{ on }">
+              <v-btn text small color="primary" id="start-menu-btn" v-on="on">
+                <v-icon id="start-menu">{{dropdown ? 'mdi-menu-up' : 'mdi-menu-down'}}</v-icon>
+              </v-btn>
+            </template>
+            <v-btn text color="primary" id="more-changes-btn"
+              @click="onChangeClicked(); dropdown = false">
+              <v-icon small color="primary">mdi-pencil</v-icon>
+              <span>{{ editLabel }}</span>
+            </v-btn>
+          </v-menu>
+        </span>
+
+      </v-col>
+    </v-row>
+  </section>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop, Watch } from 'vue-property-decorator'
+import { Getter, Action } from 'vuex-class'
+import { CommonMixin, DateMixin } from '@/mixins/'
+import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-picker'
+import { ActionBindingIF } from '@/interfaces'
+
+@Component({
+  components: {
+    DatePickerShared
+  }
+})
+export default class ResolutionDate extends Mixins(CommonMixin, DateMixin) {
+  // Global getters
+  @Getter isBenCorrectionFiling!: boolean
+  @Getter hasResolutionDateChanged!: boolean
+  @Getter getCorrectionStartDate!: string
+  @Getter getBusinessFoundingDate!: string // actually date-time
```

Comment:
`// actually date-time` maybe the format would help explain here?

Reply:
I believe this comment was taken from the state variable itself. Perhaps the getter could be renamed to `getBusinessFoundingDatetime` ?